src/class/audio/audio_device.c

```diff
@@ -1424,16 +1433,16 @@ static bool audiod_set_interface(uint8_t rhport, tusb_control_request_t const *
   // 3. Open new EP
 
   uint8_t const itf = tu_u16_low(p_request->wIndex);
-  uint8_t const alt = tu_u16_low(p_request->wValue);
+  volatile uint8_t const alt = tu_u16_low(p_request->wValue);
```

Comment:
`volatile` should be removed, it is redundant

Reply:
Sorry for that, left from debugging!

Reply:
no problem at all :)